src/core/js/router.js

```diff
@@ -246,7 +246,7 @@ class Router extends Backbone.Router {
   setGlobalClasses() {
     const currentModel = Adapt.location._currentModel;
 
-    const htmlClasses = (currentModel && currentModel.get('_htmlClasses')) || '';
+    const htmlClasses = currentModel?.get('_htmlClasses') || '';
```

Comment:
Possible argument for using nullish coalescing operator here - just on the off-chance someone decides they want to use "false" as an HTML class?

Reply:
Ignore me, "false" wouldn't actually be considered 'falsy' would it?